<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Найти и заменить</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Что:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Чем:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="whatLineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="byWhatLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="wholeWord">
       <property name="text">
        <string>Только слово целиком</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="matchCase">
       <property name="text">
        <string>С учётом регистра</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QPushButton" name="replaceButton">
       <property name="text">
        <string>Заменить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>Найти далее</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="replaceAllButton">
       <property name="text">
        <string>Заменить все</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
